templates/tools/dockerfile/python_stretch.include

```diff
@@ -1,9 +1,9 @@
-FROM debian:stretch
+FROM debian:bullseye
```

Comment:
this file is called "python_stretch.include", so you can't just change it into bullseye without renaming it.

Reply:
I can't argue with that logic. 😆